lib/board.rb

```diff
@@ -0,0 +1,66 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+  def initialize
+      reset!
+    end
+
+  def self.cells
+    @cells
```

Comment:
With `attr_accessor :cells`, this method is not needed, and it looks like it is not actually used in your code. `attr_accessor` is a macro that automatically adds both the setter method and the getter method to your class for the selected attribute. In this case, both
```
def cells
  @cells
end
```
and
```
def cells=(cells)
  @cells = cells
end
```
have been included in your class through the `attr_accessor`.